src/api/api.js

```diff
@@ -0,0 +1,11 @@
+const BASE_URL = 'http://www.omdbapi.com/?apikey=5ba996a5&t=';
```

Comment:
```suggestion
const BASE_URL = 'https://www.omdbapi.com/?apikey=5ba996a5&t=';
```
Without it your app will not work properly on github pages